### PR TITLE
followups refactor

### PR DESCRIPTION
**Pull request by aaronshiel**
The follow ups widget was once a component that lived in the record page. I moved it to its own page with its own hooks.

### CODE REVIEW

**Review by beatthat — COMMENTED** [review]
question: everywhere else we do array params this style: `?mentor=m1&mentor=m2`

...what drives doing them comma separated now?

**Comment by aaronshiel** [comment]
> question: everywhere else we do array params this style: `?mentor=m1&mentor=m2`
> 
> ...what drives doing them comma separated now?

You can specify a parameter `arrayFormat` to queryString.parse() (which I did in use-with-location.tsx)

I wasn't aware of the format you specified.

**Comment by beatthat** [comment]
> > question: everywhere else we do array params this style: `?mentor=m1&mentor=m2`
> > ...what drives doing them comma separated now?
> 
> You can specify a parameter `arrayFormat` to queryString.parse() (which I did in use-with-location.tsx)
> 
> I wasn't aware of the format you specified.

wondering if would be hard to make it work the other way? Just because we do it that way everywhere else (e.g. multiple mentors for a mentor panel and list of recommended questions)

File: cypress/cypress/integration/followups.spec.ts [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+  mentorType: MentorType.CHAT,
+  lastTrainedAt: null,
+  subjects: [
+    completeSubject({
+      _id: "background",
+      name: "background",
+      categories: [{ id: "cat", name: "cat", description: "cat" }],
+      questions: [
+        completeSubjectQuestion({
+          question: completeQuestion({
```
Comment (kycarr) [inline cypress/cypress/integration/followups.spec.ts:37]: The way questions are fetched has changed somewhat, so subject only needs the _id and not the other stuff for questions:

questions: [completeSubjectQuestion({category: {id: "cat"}, topics: [], question: {_id: "A1_1_1}})]

File: cypress/cypress/integration/followups.spec.ts [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+            type: QuestionType.QUESTION,
+            paraphrases: [],
+          }),
+          category: { id: "cat" },
+          topics: [],
+        }),
+        completeSubjectQuestion({
+          question: completeQuestion({
+            _id: "A2_1_1",
+            question: "How old are you now?",
```
Comment (kycarr) [inline cypress/cypress/integration/followups.spec.ts:50]: can remove question field here

File: cypress/cypress/integration/followups.spec.ts [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+        }),
+      ],
+    }),
+    completeSubject({
+      _id: "repeat_after_me",
+      questions: [
+        completeSubjectQuestion({
+          question: completeQuestion({
+            _id: "A3_1_1",
+            question:
```
Comment (kycarr) [inline cypress/cypress/integration/followups.spec.ts:62]: can remove question field from question here

File: cypress/cypress/integration/followups.spec.ts [hunk 4]
```diff
@@ -0,0 +1,393 @@
...
+        }),
+      ],
+    }),
+  ],
+  answers: [
+    {
+      _id: "A1_1_1",
+      question: completeQuestion({
+        _id: "A1_1_1",
+        question: "Who are you and what do you do?",
```
Comment (kycarr) [inline cypress/cypress/integration/followups.spec.ts:95]: Same here, question in answer only needs _id (remove question text field)

**Review by kycarr — CHANGES_REQUESTED** [review]

**Review by kycarr — APPROVED** [review]

**Pull request merged**